### ISSUE TITLE
Is this package dead?

### ISSUE THREAD

**Issue by cjancsar** [flexxnn/sequelize-auto-migrations#61]
There are PRs open from 2017....

Is there an actively maintained fork?

**Comment by cjancsar** [comment]
@flexxnn if you are not maintaining this library, what would it take for me to become a contributor/maintainer?

**Comment by karltaylor** [comment]
@cjancsar You could always fork it and improve

**Comment by cjancsar** [comment]
@karltaylor that creates a disparity in the community. It is better to focus on maintaining one library then forking...

**Comment by leonelpena** [comment]
I agree with @cjancsar. I am interested on maintaining this library and I was also thinking of forking since there is no answer from @flexxnn.

**Comment by karltaylor** [comment]
> @karltaylor that creates a disparity in the community. It is better to focus on maintaining one library then forking...

It is always better but not always possible due to unmanaged repositories, it would be good if @flexxnn could open this up to invite collaborators, or if that fails, fork to create a new main repo and direct people there via issues like this 😊

**Comment by moda20** [comment]
is there an active fork of this repo ?

**Comment by mgbiedma** [comment]
It seems that @flexxnn is no longer supporting this package. He has no activity in Github since August (and he used to be quite an active member).

This package does not support Sequelize v5 out of the box and Sequelize v6 is already out as a beta.

I support @cjancsar 's proposal. Maybe it's the time to fork and maintain/improve..

**Comment by MRVMV** [comment]
you can see this maintained one :)
[sequelize-mig](https://github.com/MRVMV/sequelize-mig)

**Comment by matmar10** [comment]
To anyone still actively using `sequelize-auto-migrations`, I've made a fork and published it as `sequelize-lazy-migrations`. 

I plan on adding some baseline unit tests, and - once we have a baseline - merging as many PRs as possible opened against this repository. 

I'd merge all PRs right away but I am wary of breaking changes without a proper unit test baseline.
